.github/workflows/ci-build.yaml

```diff
@@ -37,27 +37,32 @@ jobs:
       - uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd  # v3.0.1
 
   build:
-    name: "build ${{ matrix.name-prefix }} (py ${{ matrix.python-version }} on ${{ matrix.os }}, x64=${{ matrix.enable-x64}})"
-    runs-on: ${{ matrix.os }}
+    name: "build ${{ matrix.name-prefix }} (py ${{ matrix.python-version }} on ubuntu-20.04-16core, x64=${{ matrix.enable-x64}})"
```

Comment:
I assume you'll update the name in a seperate change? (It needs a copybara change since it has a status check on this name)